contrib/hawq-docker/centos6-docker/hawq-test/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ -z "${NAMENODE}" ]; then
+  export NAMENODE=${HOSTNAME}
+fi
+
+if [ ! -f /etc/profile.d/hadoop.sh ]; then
+  echo '#!/bin/bash' | sudo tee /etc/profile.d/hadoop.sh
+  echo "export NAMENODE=${NAMENODE}" | sudo tee -a /etc/profile.d/hadoop.sh
+  sudo chmod a+x /etc/profile.d/hadoop.sh
+fi
+
+sudo start-hdfs.sh
+sudo sysctl -p
+sudo ln -s /usr/lib/libthrift-0.9.1.so /usr/lib64/libthrift-0.9.1.so
```

Comment:
This link can only be done at this time  ? If not , we could move it to the dockerfile after install libthrift. If we move it, this entrypoint shell script is the same with the entrypoint.sh under centos7-docker.